cmd/finch/virtual_machine_local.go

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit_ virtual_machine_native.go

Reply:
how is this used? if the file is not required can we do a arch selective check wherever its called?

Reply:
it's just a stub to avoid branching and fragmenting `main.go` even further